### DISCUSSION TITLE
Can't seem to get it to work.

### DISCUSSION THREAD

**Discussion by trevorwrightdev** [jeffreylanters/react-unity-webgl#326] [Question and Answer]
Sorry, I'm new to web development and I can't seem to get this module to work. Here is my code for my "Game" component" 
<img width="607" alt="image" src="https://user-images.githubusercontent.com/86077703/153518865-8fef2ab7-9bf0-4793-a257-df5edf47cd64.png">
And here is the component on the page:
<img width="424" alt="image" src="https://user-images.githubusercontent.com/86077703/153518913-cfd3bd4c-898d-4449-990d-76eaa1b50627.png">
But when I test it out I just get a black screen. Can anyone help me out? Thank you.

**Comment by jeffreylanters** [comment]
The URLs are public, strip the first "../public" from them and try again.

**Reply by trevorwrightdev** [reply]
Thank you! That worked.
